packages/web-pkg/src/composables/piniaStores/theme.ts

```diff
@@ -0,0 +1,149 @@
+import merge from 'deepmerge'
```

Comment:
what's the benefit of using `deepmerge` instead of lodash `merge`?

Reply:
As discussed: let's leave it for now because we already ship the `deepmerge` dependency.